saga/seata-saga-engine-store/src/main/java/io/seata/saga/engine/serializer/impl/ExceptionSerializer.java

```diff
@@ -38,24 +39,14 @@
 
         byte[] result = null;
         if (o != null) {
-            ObjectOutputStream oos = null;
-            try {
-                ByteArrayOutputStream baos = new ByteArrayOutputStream();
-                oos = new ObjectOutputStream(baos);
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            try (ObjectOutputStream oos = new ObjectOutputStream(baos)) {
```

Comment:
if ObjectOutputStream close error , it will throw an exception?

Reply:
yes,catch exceptions